src/components/Map/Map.vue

```diff
@@ -380,7 +377,7 @@ onMounted(() => {
     .on("mouseleave", UNCLUSTERED_LAYER_ID, function () {
       map.getCanvas().style.cursor = "";
     })
-    .on("styledata", () => {
+    .on("styledata", (event) => {
```

Comment:
Are you consuming this somewhere or is adding the param just for future clarity?

Reply:
Good spot. I was logging it out when troubleshooting to see what was triggering the styledata events. I'll remove.